resource-recorder/src/dal.rs

```diff
@@ -177,6 +180,18 @@ impl Dal for Sqlite {
         Ok(result)
     }
 
+    async fn get_resource(&self, resource: &Resource) -> Result<(), DalError> {
+        sqlx::query("DELETE FROM resources WHERE project_id = ? AND service_id = ? AND type = ?")
+            .bind(resource.project_id.map(|u| u.to_string()))
+            .bind(resource.service_id.map(|u| u.to_string()))
+            .bind(resource.r#type)
+            .execute(&self.pool)
+            .await
+            .map(|_| ())?;
+
+        Ok(())
+    }
```

Comment:
I think you missed a copy-paste here, we need to rewrite this to rather get a specific resource.